### DISCUSSION TITLE
Turn device on/off in x minutes

### DISCUSSION THREAD

**Discussion by Xitee1** [home-assistant/frontend#13523] [Dashboard]
It would be really helpful if a long-press on a toggle could be integrated which then opens a little menu where you can set a timer to turn on/off the device in a specific amount of time.
I know that this can be done with automations and scripts, but I mean to just set a custom time on the go. For example, if I go to bed and want that the climate turns off in 15 minutes that I just have to long-press the off button and set a timer to 15 minutes. And don't have to quickly create a script with a delay just for this when I'm tired and want to sleep.

**Comment by KNXBroker** [comment]
I also would love to have this for my pool. I have created a timer helper for my pump with 2h hours, but sometimes I want to get it operating for more than two hours. 

For the time being I have to modify the timer setting, and revert it afterwards. 

I would appreciate a slider in the timer entity more info menu for temporary override.
